rpcservers/neutrino/neutrino_server.go

```diff
@@ -188,16 +191,16 @@ func (r *ServerShell) CreateSubServer(configRegistry lnrpc.SubServerConfigDispat
 // of the neutrino node.
 //
 // NOTE: Part of the NeutrinoServer interface.
-func (s *Server) Status(ctx context.Context,
-	in *StatusRequest) (*StatusResponse, error) {
+func (s *Server) Status(ctx context.Context, in *neutrinorpc.StatusRequest) (
```

Comment:
nit: break before `in` instead to avoid dangling open parenthesis at line end. See https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md#wrapping-long-function-definitions for reference (best effort, where it's possible).

Reply:
Might I sugest to clarify the formatting document to specify "avoid breaking parameter or return bracketed groups at the beginning or end, so as to leave no dangling open/close brackets?" or is it just the open brackets that are unclear. I did what I did to satisfy the requirements as they stand currently.

Personally, I favor the one parameter/return value per line if over 2 and past one line. The convention you are requiring here breaks the visual flow of the groups. But I will change them all to fit your request.

Reply:
I get it that the intention is to avoid the close bracket at the start of the line... because gofmt puts that back at column zero. This one point by itself would obviate much of the examples.

I would suggest make the rule to put a parameter on the next line:

- if it extends no more than 16 characters over 80
- or the open bracket of the parameters would end up on the start of a line, the above rule extends to 19.

This essentially is the rule, just have to keep it in mind for a while before it becomes automatic.

Reply:
Those sound like great additions to the `code_formatting_rules.md` file. A PR would be very welcome to make the rules clearer!